installer/omnisharp-manager.ps1

```diff
@@ -52,5 +52,27 @@ if (Test-Path -Path $location) {
 }
 New-Item -ItemType Directory -Force -Path $location
 
+#Run as SilentlyContinue to avoid progress bar that can't be seen
+$ProgressPreference = 'SilentlyContinue'
 Invoke-WebRequest -Uri $url -OutFile $out
-Expand-Archive $out -DestinationPath $location -Force
+
+#Run Expand-Archive in versions that support it
+if ($PSVersionTable.PSVersion.Major -gt 4)
+{
+    Expand-Archive $out -DestinationPath $location -Force
+}
+else
+{
+    Add-Type -AssemblyName System.IO.Compression.FileSystem
+    [System.IO.Compression.ZipFile]::ExtractToDirectory( $out, $location )
+}
+
+#Check for file to confirm download and unzip were successful
+if(Test-Path -path "$($location)\OmniSharp.Roslyn.dll")
+{
+    Write-Output "OmniSharp Roslyn Server installed at '$($location)'"
+}
+else
+{
+    Write-Output "Failure"
```

Comment:
Since you have added different exit routes, it would be particularly handy to also [return exit codes](https://weblogs.asp.net/soever/returning-an-exit-code-from-a-powershell-script). It would be even better if it could use the same exit codes as POSIX Shell, `0` for success; `1` for failed (but I'm not sure if this is even possible, as I don't know much about PowerShell).

Reply:
That sounds great, I will look into adding that. I originally went with returning strings because I wanted to send back the verified location of the install to the vim script instead of the vim script assuming it was installed at the location it had sent. I also don't know much about PowerShell and it returns a whole lot more data that I wasn't sure how to parse in vimscript. Since my intent of doing it this way didn't work out anyway, this makes more sense.

Reply:
I did find out how to do the exit codes and I changed over the console output to match this method as well. The strings weren't doing what I meant them to do so the 0 and 1 success codes make more sense.